Makefile

```diff
@@ -271,26 +237,12 @@ run-coredns: stop-coredns
 stop-coredns:
 	-docker rm -f coredns
 
+st:
```

Comment:
I missed where will this be used.

Reply:
The `tests` target in Makefile.common requires a `st` target.